packages/about-you/api-client/tests/api/clearCart/index.spec.ts

```diff
@@ -0,0 +1,27 @@
+import clearCart from '../../../src/api/clearCart';
+import { apiClient } from '../../../src/index';
+
+describe('[about-you-api-client] clearCart', () => {
+  it('adds item to cart', async () => {
+    const mockGetBasket = jest.fn(() => Promise.resolve({
+      basket: {
+        items: [
+          {
+            key: 'test'
+          }
+        ]
+      }
+    }));
+
+    const mockClearCart = jest.fn((params) => params);
+
+    // @ts-ignore
+    apiClient.basket.get = mockGetBasket;
```

Comment:
This isn't really how you should mock apiClient. I suggest to use jest.mock instead.